microsoftgraph/microsoft-graph-explorer-v4
# Task: Reorder revoking permission event messages

## Overview
This task reorders revoking permission events

### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes
The revoking permission initialization takes some time as there are a couple of async calls to get tenant information and permission grants.
This PR reorders the events so that they are dispatched before the initialization process begins. 

## Testing Instructions

* How to test this PR
Sign in with an AAD account
Consent to any permission
Click on Unconsent
Notice that the event messages are displayed much faster
